sfa_dash/templates/forms/admin/permission_macros.jinja

```diff
@@ -20,7 +20,7 @@
 {% autoescape true %}
 {% set uuid = row_values['site_id'] %}
 <tr visible="true">
-  <td><input type="checkbox" name="objects-list-{{ index }}" value="{{ uuid }} {% if checked %}checked{% endif %}"></td>
+  <td><input type="checkbox" name="objects-list-{{ index }}" value="{{ uuid }}" {% if checked %}checked{% endif %}"></td>
```

Comment:
stray " at end?

Reply:
Yeah, thanks this is probably what caused the issue to begin with.